### PR TITLE
Add Vagrant, z.lua and Droidcam

### PR DESCRIPTION
**Pull request by zkyq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -615,6 +615,7 @@
 - [![Open-Source Software][OSS Icon]](https://github.com/vkohaupt/vokoscreen) [Vokoscreen](http://linuxecke.volkoh.de/vokoscreen/vokoscreen.html) - A free, multilingual and easy to use screencast recorder with audio for Linux. It has many features.
 
 #### Screenshot
+- [![Open-Source Software][OSS Icon]](https://github.com/linuxdeepin/deepin-screenshot) [deepin-screenshot](https://www.deepin.org/en/original/deepin-screenshot) - Deepin Screenshot is characterized with smart window identification, shortcuts supporting, image editing, delay screenshot, social sharing, smart saving, image resolution adjusting and other functions.
```
Comment (luong-komorebi) [inline README.md:618]: The Github link is archived and I can see the project being migrated. Could you please help by either removing it or update with the latest location ( I cant seem to find one )

File: README.md [hunk 2]
```diff
@@ -929,6 +930,7 @@
 - [![Open-Source Software][OSS Icon]](https://convertall.bellz.org/download.html) [Convertall](https://sourceforge.net/projects/convertall/) - A program that can convert many units of measurement to other units.
 - [![Open-Source Software][OSS Icon]](https://github.com/chamfay/Curlew) [Curlew](https://github.com/chamfay/Curlew) - A GTK media converter for the GNOME desktop.
 - [![Open-Source Software][OSS Icon]](https://git.suckless.org/dmenu/) [dmenu](https://tools.suckless.org/dmenu/) - dmenu is a dynamic menu for X, originally designed for dwm. It manages large numbers of user-defined menu items efficiently.
+- [![Open-Source Software][OSS Icon]](https://github.com/aramg/droidcam/tree/master/linux)[Droidcam](https://www.dev47apps.com/) - DroidCam turns your Android device into a wireless webcam for your PC.
```
Comment (luong-komorebi) [inline README.md:933]: 👍

File: README.md [hunk 3]
```diff
@@ -1023,9 +1025,11 @@
 - [Top 10 command line tools](https://lifehacker.com/top-10-command-line-tools-399468)
 - [![Open-Source Software][OSS Icon]](https://github.com/soimort/translate-shell) [translate-shell](https://www.soimort.org/translate-shell) - Command-line translator using Google Translate, Bing Translator, Yandex.Translate, etc.
 - [![Open-Source Software][OSS Icon]](https://github.com/jml/undistract-me) [undistract-me](https://github.com/jml/undistract-me) - A command line program that plays a sound or sends a notification when a long command has finished running in the command line.
+- [![Open-Source Software][OSS Icon]](https://github.com/hashicorp/vagrant) [vagrant](https://www.vagrantup.com/) - Vagrant is a tool for building and distributing development environments.
```
Comment (luong-komorebi) [inline README.md:1028]: Greate one. As a developer, I feel bad not having this on the list sooner.

File: README.md [hunk 4]
```diff
@@ -1023,9 +1025,11 @@
...
 - [wicd-curses](https://wiki.archlinux.org/index.php/wicd#Running_Wicd_in_Text_Mode) - Command line WiFi connection manager.
 - [![Open-Source Software][OSS Icon]](https://github.com/BurntSushi/xsv) [xsv](https://github.com/BurntSushi/xsv) - A fast CSV command line toolkit written in Rust.
 - [![Open-Source Software][OSS Icon]](https://github.com/rg3/youtube-dl) [youtube-dl](https://rg3.github.io/youtube-dl/) - youtube-dl is a command-line program to download videos from YouTube.com and a few more sites. It requires the Python interpreter (2.6, 2.7, or 3.2+), and it is not platform specific.
+- [![Open-Source Software][OSS Icon]](https://github.com/skywind3000/z.lua) [z.lua](https://github.com/skywind3000/z.lua) - A new cd command that helps you navigate faster by learning your habits.
```
Comment (luong-komorebi) [inline README.md:1032]: Currently using this, too

File: README.md (hunk 1 above)
Comment (zkyq) [reply README.md]: Yes, deepin-screenshot has migrated into deepin-screen-recorder.
I don’t use it now, flameshot instead.

**Pull request merged**